api/v1alpha1/release_types.go

```diff
@@ -138,128 +114,128 @@ type Release struct {
 	Status ReleaseStatus `json:"status,omitempty"`
 }
 
-// HasStarted checks whether the Release has a valid start time set in its status.
-func (r *Release) HasStarted() bool {
-	return r.Status.StartTime != nil && !r.Status.StartTime.IsZero()
+// HasDeploymentFinished checks whether the Release has been successfully deployed via GitOps.
+func (r *Release) HasDeploymentFinished() bool {
+	condition := meta.FindStatusCondition(r.Status.Conditions, deployedConditionType.String())
+	return condition != nil && condition.Status != metav1.ConditionUnknown
 }
 
-// HasSucceeded checks whether the Release has succeeded or not.
-func (r *Release) HasSucceeded() bool {
-	return meta.IsStatusConditionTrue(r.Status.Conditions, releaseConditionType)
+// HasReleaseFinished returns a boolean indicating whether the Release's status indicates that it is done or not.
```

Comment:
this would also become `isXed` when we decide on X

Reply:
Processed

Reply:
Old comment.